fern/pages/instruments-guide/futures/tier-2/paxg-usd-perp.mdx

```diff
@@ -9,9 +9,9 @@ export const fundingPeriod = "8 hours";
 export const tickSize = "10 USD";
 export const orderSizeIncrement = "0.001 PAXG";
 export const minOrderValue = "50 USD";
-export const maxOrderSize = "68,000 PAXG";
+export const maxOrderSize = "89 PAXG";
 export const maxOpenOrders = "50";
-export const positionLimit = "220,000 PAXG";
+export const positionLimit = "290 PAXG";
```

Comment:
@varunjain15 - this seem's odd :thinking:  can we confirm these values are good?

Reply:
yes checked with Kiet!
(these are also updated on [specs](https://docs.google.com/spreadsheets/d/1pfDvliW8pBQY6rSghyHDH-mM6R3ZC6J8gFvjERmImMo/edit?gid=1946449001#gid=1946449001) now)